### ISSUE TITLE
node_modules/angular-archwizard/archwizard.css MISSING

### ISSUE THREAD

**Issue by fredvaliant** [madoar/angular-archwizard#209]
Missing css

![image](https://user-images.githubusercontent.com/18463251/56895539-f9917980-6abb-11e9-9f07-ffcf25961ce3.png)

**Comment by fernandezjose** [comment]
Same here...

**Comment by madoar** [comment]
The `archwizard.css` file is currently part of the development branch and not the master (release) branch. The next version of `angular-archwizard` (most will most likely become version `5.0.0`) will contain the generated `archwizard.css` file.

I expect to release the next version of `angular-archwizard` in the next two weeks.

**Comment by ste7e0** [comment]
@madoar so what we can do to use angular-archwizard?? just wait??

**Comment by madoar** [comment]
Alternatively you can build the development branch on your own and include your local build in your application until version `5.0.0` has been released.

`angular-archwizard` can be built using `npm run build`. Afterwards you will find the build files inside the generated `dist` folder.

**Comment by satnar** [comment]
Is the 5.0.0 delayed inordinnately? I adopted to this component after lot of read and code efforts in my project. The wizard is spewing all fields and steps in the same window... somehow troubleshooted to identify this missing CSS file. Its a pity that all your great work is unusable in lieu of this issue.

**Comment by timvandesteeg** [comment]
https://github.com/madoar/angular-archwizard/tree/develop/src/css

As a temporary fix, simply add these scss files to assets folder and include them in your angular.json.
`"path/to/your/assets/css/archwizard.scss"`
Instead of
`"node_modules/angular-archwizard/archwizard.scss"
`

**Comment by madoar** [comment]
Version `5.0.0` is currently blocked by a visual bug of the updated stylesheets described in #213.
I have some ideas how to solve the issue but I haven't yet decided which solution to is best long-term. Therefore I'm currently waiting for @earshinov to get back before deciding how to fix the issue. I expect to hear from @earshinov around the end of next week.

**Comment by EfkanKnz** [comment]
We are still waiting... Can you tell us when it's done?

**Comment by madoar** [comment]
We're almost there. After merging #221 we should be good to go for the next release.

**Comment by earshinov** [comment]
@madoar , What about #182 ? I think that it should better be fixed before the release, especially because the fix might involve changes in the publicly visible API of `NavigationMode` classes.

**Comment by madoar** [comment]
I would like to have it fixed before, but I still don't have a good idea how to fix this elegantly

**Comment by madoar** [comment]
I have just released version `5.0.0`! `angular-archwizard` version `5.0.0` comes with a separate css file containing all defined css styles of the wizard. Feel free to give it a try!

If there are any issues or questions feel free to open a new issue
